### PR TITLE
Update packages in customer account template

### PR DESCRIPTION
**Pull request by lihaokx**
### Background

Change customer account template:
1. ui extension package
2. Import new package

### Solution

(Describe your solution, why this approach was chosen, and what the alternatives/impacts may be)

### Checklist

- [x] I have :tophat:'d these changes
- [x] I have squashed my commits into chunks of work with meaningful commit messages

### CODE REVIEW

File: customer-account-extension/src/CustomerAccountExtension.liquid [hunk 1]
```diff
@@ -1,25 +1,35 @@
...
+  export default reactExtension(
+    'customer-account.dynamic.render',
+     () => <CustomerAccountExtension />
+  );
+
+  function CustomerAccountExtension() {
+    const translate = useTranslate();
+
+    return (
+      <Banner title="{{ name }}">
```
Comment (brianshen1990) [inline customer-account-extension/src/CustomerAccountExtension.liquid:15]: will this be replaced with shop name?

Comment (lihaokx) [reply customer-account-extension/src/CustomerAccountExtension.liquid:15]: No. Should be replaced by app name

Comment (brianshen1990) [reply customer-account-extension/src/CustomerAccountExtension.liquid:15]: not by us right? but liquid will fill it automatically?

Comment (lihaokx) [reply customer-account-extension/src/CustomerAccountExtension.liquid:15]: Yes, you are right.

File: customer-account-extension/src/CustomerAccountExtension.liquid [hunk 2]
```diff
@@ -1,25 +1,35 @@
...
+    );
   }
+
   {%- else -%}
-  import { extend, Banner } from "@shopify/customer-account-ui-extensions";
-  
-  extend('customer-account.dynamic.render', (root, api) => {
+  import { extension, Banner } from "@shopify/ui-extensions/customer-account";
+
+  export default extension("customer-account.dynamic.render", (root, api) => {
```
[the remark this answers is not in the document]
Comment (lihaokx) [reply customer-account-extension/src/CustomerAccountExtension.liquid]: done

**Review by brianshen1990 — APPROVED** [review]

File: customer-account-extension/src/CustomerAccountExtension.liquid [hunk 3]
```diff
@@ -1,25 +1,35 @@
 {%- if flavor contains "react" -%}
   import React from 'react';
-  import {render, Banner} from '@shopify/customer-account-ui-extensions-react';
-  
-  render('customer-account.dynamic.render', (api) => <App api={api} />);
-  
-  function App(api) {
-    return <Banner>{api.i18n.translate("welcome")}</Banner>;
+  import { Banner } from '@shopify/ui-extensions-react/customer-account';
+  import{ reactExtension, useTranslate } from '@shopify/ui-extensions-react/customer-account';
```
Comment (vividviolet) [inline customer-account-extension/src/CustomerAccountExtension.liquid:4]: I think these 2 imports should be combined

File: customer-account-extension/src/CustomerAccountExtension.liquid [hunk 4]
```diff
@@ -1,25 +1,35 @@
...
+
+  export default extension('customer-account.dynamic.render', (root, api) => {
     const { i18n } = api;
 
     root.appendChild(
       root.createComponent(
         Banner,
-        {},
-        i18n.translate("welcome")
+        { title: "{{ name }}" },
```
Comment (vividviolet) [inline customer-account-extension/src/CustomerAccountExtension.liquid:30]:
```suggestion
        { title: '{{ name }}' },
```

**Review by vividviolet — APPROVED** [review]

**Pull request merged**